bin/i18n/i18n_script_utils.rb

```diff
@@ -321,4 +325,44 @@ def self.fix_yml_file(filepath)
 
     File.write(filepath, yml_data)
   end
+
```

Comment:
This whole method has been moved from the sync-out script. Now we can use it in other modules calling `I18nScriptUtils.file_changed?(locale, file)`